CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 BREAKING CHANGES:
 
 FEATURES:
+* Metrics: add support for metrics in Consul. This enables support for Consul Agent metrics,
+  Consul Gateway metrics, metrics merging to serve both application and sidecar metrics and support to configure a metrics provider for the Consul UI.
+  Additionally, adds templates for a demo installation of Prometheus and Grafana.
```

Comment:
Can we add that metrics merging will only work with Consul 1.10.0-alpha1 and higher?

Reply:
Made the change.